go/identify3/identify3_test.go

```diff
@@ -0,0 +1,251 @@
+package identify3
+
+import (
+	"github.com/keybase/client/go/externalstest"
+	"github.com/keybase/client/go/kbtest"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	insecureTriplesec "github.com/keybase/go-triplesec-insecure"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+	"sync"
+	"testing"
+)
+
+func SetupTest(tb libkb.TestingTB, name string) libkb.TestContext {
+	tc := externalstest.SetupTest(tb, name, 2)
+	// use an insecure triplesec in tests
+	tc.G.NewTriplesec = func(passphrase []byte, salt []byte) (libkb.Triplesec, error) {
+		warner := func() { tc.G.Log.Warning("Installing insecure Triplesec with weak stretch parameters") }
+		isProduction := func() bool {
+			return tc.G.Env.GetRunMode() == libkb.ProductionRunMode
+		}
+		return insecureTriplesec.NewCipher(passphrase, salt, warner, isProduction)
+	}
+	return tc
+}
```

Comment:
I tried this locally and it failed to compile:

    > go test
    # github.com/keybase/client/go/identify3
./identify3_test.go:23:37: not enough arguments in call to triplesec.NewCipher
	have ([]byte, []byte, func(), func() bool)
	want ([]byte, []byte, triplesec.Version, func(), func() bool)
    FAIL	github.com/keybase/client/go/identify3 [build failed]

Reply:
thx, rebase bug, working on it now!